knife-supermarket.gemspec

```diff
@@ -14,5 +14,5 @@ Gem::Specification.new do |s|
   s.homepage    = 'https://github.com/cwebberOps/knife-supermarket'
   s.license       = 'Apache 2.0'
   s.require_paths = ["lib"]
-  s.add_dependency "chef", ">= 0.10.10"
+  s.add_dependency "chef", ">= 0.10.10", "<= 12.11.18"
```

Comment:
I don't think checking for `defined?(Chef::Knife::SupermarketDownload)`—or other commands—is going to work for the deprecation notice. Those classes weren't _removed_. They were relocated from the `knife-supermarket` gem to the [`chef` gem](https://github.com/chef/chef/blob/master/lib/chef/knife/supermarket_download.rb#L24) and reduced to aliases of the equivalent `cookbook site` commands, right?